docs/source/examples/tex.rst

```diff
@@ -1,6 +1,27 @@
 LaTeX
 =================================
 
+The Tex() mobject
++++++++++++++++++++
+Just as you can use `Text()` to add text to your videos, you can use `Tex()` to insert LaTeX.
+
+.. manim:: ExampleLaTeX
+    :save_last_frame:
+
+    class ExampleLaTeX(Scene):
+        def construct(self):
+            tex = Tex(r'\LaTeX').scale(3)
+            self.add(tex)
+
+Note that we are using a raw string (r'---') instead of a regular string ('---').
+This is because TeX code uses a lot of special characters - like `\\` for example - 
+that have special meaning within a regular python string. An alternative would have
+been to write `\\\\` as in Tex('\\\\LaTeX').
```

Comment:
I think this isn't rendering the way you intended it.

Reply:
that's the weird thing, it is rendering correctly on my system at home...

Reply:
This paragraph needs some literals (double backtick blocks) or so.

+1 for the recommendation to use raw strings.